packages/devtools-source-map/src/tests/source-map.js

```diff
@@ -0,0 +1,33 @@
+const fs = require("fs");
+const path = require("path");
+
+jest.mock("devtools-network-request");
+const { getOriginalURLs } = require("../source-map");
+
+function getMap(_path) {
+  const mapPath = path.join(__dirname, _path);
+  return fs.readFileSync(mapPath, "utf8");
+}
+
+describe("source maps", () => {
+  test("getOriginalURLs", async () => {
+    const source = {
```

Comment:
Would it be possible to check in the original sources of the bundle and a script for bundling them? There could be cases where we need to fix issues with source map _generation_, so being able to regenerate test bundles in a sane way seems useful.